emop-maven-plugin/src/main/java/edu/cornell/AffectedSpecsMojo.java

```diff
@@ -108,8 +116,18 @@ private void writeMapToFile(OutputFormat format) throws MojoExecutionException {
             default:
                 try (PrintWriter writer
                              = new PrintWriter(getArtifactsDir() + File.separator + "classToSpecs.txt")) {
-                    for (Map.Entry<String, Set<String>> entry : classToSpecs.entrySet()) {
-                        writer.println(entry.getKey() + ":" + String.join(",", entry.getValue()));
+                    switch (classToSpecsContent) {
+                        case MAP:
+                            for (Map.Entry<String, Set<String>> entry : classToSpecs.entrySet()) {
+                                writer.println(entry.getKey() + ":" + String.join(",", entry.getValue()));
+                            }
+                            break;
+                        case SET:
+                        default:
+                            for (String affectedSpec : affectedSpecs) {
+                                writer.println(affectedSpec);
+                            }
+                            break;
```

Comment:
This break statement is redundant

Reply:
I am also worried that a switch withing a switch is unnecessarily complex. Move the inner one into a method, maybe?